src/utils/routeHandler.ts

```diff
@@ -0,0 +1,16 @@
+import router from "next/router";
+
+export const routeHandler = (endpoint?: string) => {
+  endpoint? router.push({
+    pathname: "courts/" + endpoint,
```

Comment:
Do you mind changing this part, making it more common?  make "courts/" a parameter.